rs/tests/ckbtc/BUILD.bazel

```diff
@@ -39,3 +52,120 @@ rust_library(
     crate_name = "ic_tests_ckbtc",
     deps = DEPENDENCIES,
 )
+
+CKBTC_RUNTIME_DEPS = [
+    # Keep sorted.
+    "//rs/bitcoin/ckbtc/kyt:kyt_canister",
+    "//rs/bitcoin/ckbtc/minter:ckbtc_minter_debug",
+    "//rs/ledger_suite/icrc1/ledger:ledger_canister",
+    "@btc_canister//file",
+]
+
+system_test_nns(
+    name = "ckbtc_minter_basics_test",
+    env = {
+        "IC_CKBTC_KYT_WASM_PATH": "$(rootpath //rs/bitcoin/ckbtc/kyt:kyt_canister)",
+        "IC_CKBTC_MINTER_WASM_PATH": "$(rootpath //rs/bitcoin/ckbtc/minter:ckbtc_minter_debug)",
+        "LEDGER_WASM_PATH": "$(rootpath //rs/ledger_suite/icrc1/ledger:ledger_canister)",
+        "BTC_WASM_PATH": "$(rootpath @btc_canister//file)",
+    },
+    extra_head_nns_tags = [],  # don't run the head_nns variant on nightly since it aleady runs on long_test.
+    flaky = True,
+    tags = [
+        "k8s",
+        "long_test",  # since it takes longer than 5 minutes.
+    ],
+    target_compatible_with = ["@platforms//os:linux"],  # requires libssh that does not build on Mac OS
+    runtime_deps =
+        GUESTOS_RUNTIME_DEPS +
+        UNIVERSAL_VM_RUNTIME_DEPS +
+        CKBTC_RUNTIME_DEPS,
+    deps = DEPENDENCIES + [":ckbtc"],
```

Comment:
I see `:ckbtc` is being added to `DEPENDENCIES` for every system-tests. I would consider just including  `:ckbtc` in `DEPENDENCIES` above.

Reply:
I see. Ok.